### ISSUE TITLE
Broken packaging with browserify

### ISSUE THREAD

**Issue by TheMarex** [perliedman/leaflet-routing-machine#325]
Using this [self-contained example](https://gist.github.com/TheMarex/89d8a5ebdbc8d2e1e65d9da093995b71) and doing `npm run build` I get the following error:

```
Error: Cannot find module './L.Routing.Itinerary' from '/home/patrick/Code/spielplatz/lrm_test/node_modules/leaflet-routing-machine/dist'
    at /home/patrick/Code/spielplatz/lrm_test/node_modules/resolve/lib/async.js:55:21
    at load (/home/patrick/Code/spielplatz/lrm_test/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/patrick/Code/spielplatz/lrm_test/node_modules/resolve/lib/async.js:92:31)
    at /home/patrick/Code/spielplatz/lrm_test/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:123:15)
```

**Comment by perliedman** [comment]
Seems this has broken in the same manner as leaflet-control-geocoder, and apparently I don't understand Browserify well enough.

Thanks for reporting, I'll look into it!

**Comment by TheMarex** [comment]
I think this line might be the culprit here: https://github.com/perliedman/leaflet-routing-machine/blob/master/package.json#L35

Changing that back to `src/L.Routing.Control.js` and adding the dependency to Leaflet works.

**Comment by perliedman** [comment]
Yeah, but I want to avoid specifying a Leaflet dependency, since that causes trouble with 0.7/1.0 of Leaflet - I want users to be able to use both without trouble.

Anyway, I've published 3.2.2 which should fix the problem (I botched up 3.2.1 :stuck_out_tongue_winking_eye: ) - please report if you still have problems with that one, and sorry about this trouble to begin with!
